sites/www.thunderbird.net/thunderbird/releases/index.html

```diff
@@ -24,35 +24,39 @@
         </a>
       </div>
     </div>
-    <div class="section-text">
+    <div class="section-text release-list">
         {% for int_version, versions in releases -%}
           <div class="release-container">
             <h2 class="release-major txt-gradient">
               {{ get_link(int_version, versions.major) }}
             </h2>
             {% if versions.minor -%}
             <div class="release-minors">
+              <ul aria-label="{{ _('Stability releases for version %(version)s'|format(version=versions.major)) }}">
```

Comment:
Adds context for screen reader users, otherwise they're getting a earful of stability versions.